samples/drivers/spi_flash/src/main.c

```diff
@@ -21,7 +21,7 @@
 #error Unsupported flash driver
 #endif
 
-#define FLASH_TEST_REGION_OFFSET 0xff000
+#define FLASH_TEST_REGION_OFFSET 0x100000
```

Comment:
We have M4 application image starting from offset 0x0008’0000 to 0x000F’FFFF. So we can not use the location 0xff000 for this test. Since the offset is hard-coded within the application I have modified it for testing. Shall I change it to DT_FLASH_AREA_STORAGE_OFFSET as in nvs sample? If not, I don't know what is the way out.

Note: I have given image-3 size wrong as 0x20000 in quick_feather.dts. I'll fix it in next commit.

Reply:
I think we can fix it like [here](https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/drivers/spi_flash/src/main.c#L29-L30)